### DISCUSSION TITLE
Good job on the performance!

### DISCUSSION THREAD

**Discussion by Julianoe** [TechbeeAT/jtxBoard#1171] [General]
I tested the app months ago (maybe more I don't remember) after learning the existence of vJournal and the possibility to edit them in jtxBoard + the fact that this app is compatible with DAVx², an app already in my process for synchronizing contacts and calendar events. I was really disappointed when I saw that the performance was, sorry to say it, horseshit. I don't have problems with my "old" BV6000 with other apps and this one was unusable.

Now I can use it! So congrats and thank you for making it more accessible to people with low end phones !

**Comment by patrickunterwegs** [comment]
Hi @Julianoe !
Thank you for the feedback! Partly this is thanks to improvements on the jetpack compose UI framework, partly I did a few things extra. There's still a bit to do, especially when it comes to the creation and updates of recurring tasks, but one step at a time 😉
